main.js

```diff
@@ -16,3 +22,66 @@ closeMenu.addEventListener('click', close);
 hambuChilds.forEach((chld) => {
   chld.addEventListener('click', close);
 });
+
+seeproject1.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+seeproject2.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+seeproject3.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+seeproject4.addEventListener('click', () => {
+  popupWindow.classList.add('popup-active');
+});
+
+// closewindow.addEventListener('click', () => {
+//   popupWindow.classList.remove('popup-active');
+// });
+
+const cardArray = [{
+  header: 'Tonic',
+  list: ['Canopy', 'Back End Dev', '2015'],
+  image: './img/popupdesk.png',
+  text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the standard dummy text ever since the 1500s',
+  listbuttons: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+  live: 'https://algerina.github.io/My_portfolio/',
+  source: 'https://github.com/algerina',
+
+},
+
+];
```

Comment:
- [ ] <b>REQUIRED:</b> The List you must have here, isn't just about popup. it must be about projects in general.
```Javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
when displaying the project, you must iterate through this list and append them to your page with javascript. and when we click on `see project` button, it should show us the details of the project we clicked on.